test/schema/me/conversation.js

```diff
@@ -55,5 +62,98 @@ describe("Me", () => {
         expect(conversation).toEqual(expectedConversationData)
       })
     })
+
+    it("sets the expand[] param when fetching messages", () => {
+      const query = `
+        {
+          me {
+            conversation(id: "420") {
+              id
+              initial_message
+              from {
+                email
+              }
+              messages {
+                snippet
+              }
+            }
+          }
+        }
+      `
+
+      const conversation1 = {
+        id: "420",
+        initial_message: "10/10 would buy",
+        from_email: "fancy_german_person@posteo.de",
+        messages: [
+          {
+            snippet: "Take my money!",
+          },
+        ],
+      }
+
+      const expectedConversationData = {
+        conversation: {
+          id: "420",
+          initial_message: "10/10 would buy",
+          from: {
+            email: "fancy_german_person@posteo.de",
+          },
+          messages: [
+            {
+              snippet: "Take my money!",
+            },
+          ],
+        },
+      }
+
+      gravity.onCall(1).returns(Promise.resolve({ token: "token" }))
+      impulse.withArgs("conversations/420", {}).returns("Wrong params")
```

Comment:
This and the similar line below are probably redundant but they might be helpful for understanding why the test exists.